docs/setting-up-and-running.asciidoc

```diff
@@ -16,6 +16,9 @@ To start APM Server, run:
 ./apm-server -e
 ----------------------------------
 
+NOTE: The `-e` <<global-flags,global flag>> enables logging to stderr and disables syslog/file output.
+Remove this flag if you've enabled logging in the configuration file.
+
```

Comment:
Not sure, but maybe you want to link to [APM Server status and logs](https://www.elastic.co/guide/en/apm/server/current/running-with-systemd.html#_apm_server_status_and_logs)

Reply:
What do you think about an extra sentence that says something like:
> For linux systems, also see [APM Server status and logs](https://www.elastic.co/guide/en/apm/server/current/running-with-systemd.html#_apm_server_status_and_logs)?